devsetup/Makefile

```diff
@@ -176,9 +176,10 @@ crc_cleanup: ## Destroys the CRC env, but does NOT clear ( --clear-cache ) the c
 	sudo update-ca-trust
 
 .PHONY: crc_scrub
-crc_scrub: crc_cleanup
+crc_scrub:
+	make crc_cleanup || true
 	rm -rf ~/.crc
-	sudo rm $(which crc)
+	sudo rm `which crc`
```

Comment:
Isn't ` `` ` deprecated in favor of `$(...)`?  At least I recall tools like `shellcheck` complaining about the former.

Reply:
Yes it is, but `$` has meaning in a `Makefile` so it doesn't work.
This was the easiest way to circumvent that, since escaping it with '\$' didn't seem to work for me.

Reply:
I wonder if that's why we use `$$` elsewhere in certain places?  For example [1].

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/884e173955b8ba6cbf182e1f564990d703a963a7/Makefile#L688